CiscoVM/CiscoVM-1.0.0/system.conf

```diff
@@ -34,17 +34,17 @@
           "field ID": "connect_ciscovm_token",
           "type": "encrypted",
           "mandatory": "true",
-          "tooltip":"Authentication token for destination service URL"
+          "tooltip": "Authentication token for destination service URL"
         },
         {
           "rate limiter": true,
           "display": "Rate limiter",
-          "unit": 6,

```

Comment:
Decreased to 1 second interval limit to have better reqeusts count distribution.